<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportFE24SSetupDialog</class>
 <widget class="QDialog" name="ReportFE24SSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>207</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblAccountNumber">
     <property name="text">
      <string>Номер реестра: </string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbTariffType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>общий по ОМС</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по выполнению СОМП и повышению АПМП</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по стимулирующим выплатам</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="gbOrgStructureInfo">
     <property name="title">
      <string>Определять подразделение</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbtnByPerson">
        <property name="text">
         <string>по врачу</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbtnByAction">
        <property name="text">
         <string>по движению</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTariffType">
     <property name="text">
      <string>Тип тарифа</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edtAccountNumber"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkPayStatus">
     <property name="text">
      <string>Тип реестра</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbPayStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportFE24SSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportFE24SSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPayStatus</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbPayStatus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
